views/package.json

```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.6.1",
-        "@oat-sa/tao-item-runner-qti": "0.15.5"
+        "@oat-sa/tao-item-runner-qti": "git://github.com/oat-sa/tao-item-runner-qti-fe#fix/CON-173/incorrect-counting-of-characters"
```

Comment:
Needs to be updated after https://github.com/oat-sa/tao-item-runner-qti-fe/pull/136 is merged and released